spec/helpers/helpers_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe TranzitoUtils::Helpers, type: :helper do
+  describe "page_title" do
+    before do
+      allow(view).to receive(:controller_name) { controller_name }
+      allow(view).to receive(:action_name) { action_name }
+      # this method is defined in application controller, stub it
+      def helper.controller_namespace
+      end
+      allow(view).to receive(:controller_namespace) { controller_namespace }
+    end
+    let(:controller_namespace) { nil }
+
+    context "home controller" do
+      let(:controller_name) { "home" }
+      let(:action_name) { "index" }
+      it "returns default" do
+        allow(TranzitoUtils::DEFAULT).to receive(:[]).with(:title_prefix).and_return("Dummy")
```

Comment:
This should test the actual default functionality.

I think it makes sense to add an extra test to check that the overrides work though!